libpod/runtime.go

```diff
@@ -472,7 +472,7 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (retErr error) {
 		// we will need to access the storage.
 		if os.Geteuid() != 0 {
 			aliveLock.Unlock() // Unlock to avoid deadlock as BecomeRootInUserNS will reexec.
-			pausePid, err := util.GetRootlessPauseProcessPidPath()
+			pausePid, err := util.GetRootlessPauseProcessPidPath(runtime.config.Engine.TmpDir)
```

Comment:
Add new new function and depracate old function, then we don't break ABI
util.RootlessPausePidPath(runtime.config.Engine.TmpDir)